admin/database/database.go

```diff
@@ -465,9 +480,27 @@ type UpdateServiceOptions struct {
 
 // Usergroup represents a group of org members
 type Usergroup struct {
-	ID    string `db:"id"`
-	OrgID string `db:"org_id"`
-	Name  string `db:"name" validate:"slug"`
+	ID          string    `db:"id"`
+	OrgID       string    `db:"org_id"`
+	Name        string    `db:"name" validate:"slug"`
+	Description string    `db:"description"`
+	CreatedOn   time.Time `db:"created_on"`
+	UpdatedOn   time.Time `db:"updated_on"`
+}
+
+// OrgMemberUsergroupRole represents an organization role of Usergroup
+type OrgMemberUsergroupRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	OrgName     string `db:"org_name"`
+	RoleName    string `db:"role_name"`
+}
+
+// ProjectMemberUsergroupRole represents a project role of Usergroup
+type ProjectMemberUsergroupRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	ProjectID   string `db:"project_id"`
+	ProjectName string `db:"project_name"`
+	RoleName    string `db:"role_name"`
 }
```

Comment:
I think these types could be replaced with one type `MemberUsergroup` to mirror `MemberUser`, which is used for both org and project memberships